log.go

```diff
@@ -76,33 +77,60 @@ type LeveledLogger struct {
 	stdoutOverride io.Writer
 }
 
+var apiKeyRegex *regexp.Regexp
```

Comment:
Try just:

``` go
var apiKeyRegex *regexp.Regexp = regexp.MustCompile(...)
```

Also, mind using either `Regexp` or `RE` as a suffix? I don't think we should mix the abbreviations "regex" and "regexp" which are very similar but different.